### PR TITLE
Systest 9779

### PR DESCRIPTION
**Pull request by kdivya153**

### CODE REVIEW

File: cypress/plugins/common.js [hunk 1]
```diff
@@ -23,41 +23,40 @@ const CONSTANTS = require('../support/constants/constants');
```
Comment (kschrief) [inline cypress/plugins/common.js:23]: Are the changes to this file required?

File: cypress/support/cypress-support/src/FireboltInvoker.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ const logger = require('../../Logger')('FireboltInvoker.js');
 
 const WEBSOCKET = 'WebSocket';
 const DEFAULT_WS_URL_PROTOCOL = 'ws://';
-const DEFAULT_WS_PORT = 9998;
+const DEFAULT_WS_PORT = 8080;
```
Comment (kschrief) [inline cypress/support/cypress-support/src/FireboltInvoker.js:25]: Has this been tested with Mock Firebolt?

File: cypress/plugins/common.js (hunk 1 above)
Comment (kdivya153) [reply cypress/plugins/common.js:23]: if supportConfig.js doesn't exist it throwing an error and stop the execution.Particularly this change is required if there is no config module(default config module)

File: cypress/support/cypress-support/src/FireboltInvoker.js (hunk 2 above)
Comment (kdivya153) [reply cypress/support/cypress-support/src/FireboltInvoker.js:25]: Yes,I've tested in MFOS by executing suite runs.

**Pull request merged**